client/package.json

```diff
@@ -9,7 +9,8 @@
 	"dependencies": {
 		"apollo-client": "^1.9.0",
 		"buffer": "^5.0.7",
-		"graphql-tag": "^2.4.2",
+		"expo": "^22.0.3",
+		"graphql-tag": "^2.5.0",
```

Comment:
whats this?

Reply:
needed to build this on linux earlier. can probably fuck it off.